helm/generate.sh

```diff
@@ -46,12 +46,12 @@ EOF
 
 function main() {
 
-    # docker-build: Build docker image with the manager.
-    docker buildx build --platform linux/amd64 -t "${IMG}" .
-
     # Ask admin to run this command for you as it requires privileges
-    # docker-push: Push docker image with the manager.
-    # docker push "${IMG}"
+    # docker-build: Build and push docker image with the manager.
+    docker buildx build --push --no-cache \
+    -t "${IMG}" \
+    --platform=linux/amd64,linux/arm64,linux/ppc64le,linux/s390x \
```

Comment:
This won't work. We should check whether sidecars are available for these architectures or not.

Reply:
Bala the manager binary from operator-framework already supports it:

https://quay.io/repository/operator-framework/helm-operator/manifest/sha256:4c1f130f386aa084efa00755bff07a3d41b76885efdb5f2e2bd33819d2083e8b

* linux | amd64
* linux | arm64
* linux | ppc64le
* linux | s390x

What do you mean by sidecars?

Reply:
Please refer https://github.com/minio/directpv/blob/master/docs/installation.md#prerequisites-1. Without these k8s sidecars, DirectPV doesn't work.

Reply:
You are right @balamurugana closing this PR then 👍